config/locales/fr.yml

```
@@ -8,10 +8,15 @@ fr:
     Ruby. Publiez vos gems instantanément et utilisez-les. Utilisez l'API pour interagir
     et trouver des informations sur les gems disponibles. Contribuez et améliorez
     ce site avec nous !
-  footer_sponsors_html:
-  footer_join_rt_html:
+  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat la communauté Ruby.
```

```suggestion
  footer_sponsors_html: RubyGems.org est rendu possible grâce à un partenariat avec la communauté Ruby.
```
If you want to keep the "greater", you can use "au sens large" or "élargie"

---

Good idea. [`2ccd27b` (#3112)](https://github.com/rubygems/rubygems.org/pull/3112/commits/2ccd27be336278d29fd44514466c7e1c0db9703b)